stories/Avatar/AvatarLinkWithTooltip.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import AvatarContainer from '../../src/components/Avatar/AvatarContainer'
+import AvatarIcon from '../../src/components/Avatar/AvatarIcon';
+import {Icon, UncontrolledTooltip} from '../../src'
+
+const AvatarLinkWithTooltip = () => {
+    const ids = ['Example1','Example2','Example3','Example4']
+    const target = i => document.getElementById(ids[i])
+    return(
+    <AvatarContainer>
+        <AvatarIcon size="xl" href id={ids[0]}>
+            <UncontrolledTooltip placement="left" target={()=> target(0)}>
```

Comment:
In the documentation the id should be enough: https://reactstrap.github.io/components/tooltips/

Why this round trip by DOM element?

```jsx
<UncontrolledTooltip placement="left" target={ids[0]}>
...
</UncontrolledTooltip>
```

should be enough, no?

Reply:
Yes, it is enough.